### ISSUE TITLE
This is an issue from the first class

### ISSUE THREAD

**Issue by michaelmusick** [Montana-Media-Arts/191_CreativeCoding#1]
This is the first issue post! Yea!!!!!

We made it in class. Hurray!

If you don't know what to say, please discuss your thoughts on the first class, or, how you think this semester may go for you? Maybe you would like to discuss what you want to get out of this class.

**Comment by LloydProductions** [comment]
I think that learning "Programming" as a whole will be just as difficult as learning any other language, due to the fact that we will have to translate what we are comfortable with to something completely foreign. Just from looking over the Commands I can already tell that learning abreviations and differenct key sequencing will cause issues without practice and attention to detail. Also I believe slowing down and taking a little more time than we may be accustomed to, may be the difference between a 10 minute search for an error or an hour search.

**Comment by lukedevour** [comment]
This semester I would like to gain a much deeper understanding of coding, and how I can use the skills that I learn in the class to diversify my own portfolio.

I would also like to get over the general anxiety towards learning something totally out of my comfort zone. 

I agree with Lloyd in that learning this will be just as hard as learning another language, if not more so. I am excited (if not a little apprehensive) at the prospect of learning something from the ground up.

**Comment by Noblekrug** [comment]
I think this class will be a lot of fun, but very challenging. I've taken two years of Mandarin, and coding already seems like an incredibly more intimidating language. However, as frustrating as just figuring out the CLI was, it was actually kind of fun. So, considering how fun, if not challenging, the first assignment was, I am excited to get into some more sophisticated stuff in this next semester.

**Comment by lindseysewell** [comment]
I'm a little bit nervous to learn coding because it's essentially a different language. I've been fairly good at learning other languages, but I know I'll have to deal with the frustration of a tiny problem messing everything up. The only experience I've had with coding was designing a video game about grapes back in middle school, with instruction of course.

**Comment by lindseysewell** [comment]
I am excited to visit the artistic aspect of coding. I can't wait to see what everyone creates.

**Comment by LloydProductions** [comment]
I agree with @lindseysewell  in seeing what artistic abilities we are able to harness and share. Coding isn't just the nicely fit puzzle pieces as we talked about in class. People like Engineers create "codes" for a certain purpose, where as I think we will be able to find something intended to be black and white and with what we learn in here we add color and use it in ways maybe an Engineers mind wouldn't have worked. 

I'm looking forward to website creation as well, as each one of us will at some point free lance in our Animation, Journalism, Digital Film careers we have to find a unique way to market and sell our abilities  to potential consumers. With a better understanding of the code I believe we will have a leg up on not using but creating media to better connect ourselves.

**Comment by DylanBelcourt** [comment]
I think that coding in general is just something that I have always wanted to be able to do. With this class, I am hyped to be able to learn a new skill that I can also use in a creative way. Although trying to learn a new language and remember everything might get the best of me, I think that this class will be a fun one.

**Comment by Marahute** [comment]
_Using_ coding language and _understanding_ coding language are two very different things. I've learned this lesson from other coding classes I've taken. It really takes a lot of repetition and ground level understanding to build that base, which I still haven't accomplished. 

Try to learn code like you would learn math, you can't skip ahead to the complicated formulas or you'll never understand the fundamentals(kind of). Just because your code is working, it doesn't mean you necessarily understand why. That's my takeaway and advice coming from other classes.

**Comment by sweber20** [comment]
I have a very minuscule preexisting knowledge of coding that I had learned from YouTube tutorials and different forums I have found online and at random. Almost all of what I have previously found have only confused me and given me more unanswered questions. I hope this class will answer a few of those questions as well as give me a steady base to develop more skills and hopefully even assist in acquiring internships.

I agree with @Marahute description of understanding code versus using code. I feel like my previous understanding of coding falls under the category of "just because your code is working, doesn't mean you necessarily understand why".

**Comment by Marahute** [comment]
@lukedevour 
"I would also like to get over the general anxiety towards learning something totally out of my comfort zone."
That's an awesome way to look at it Luke. I'm also anxious about throwing myself into another subject where I don't have a lot of knowledge. We just need to have the humility to ask lots of questions I think.

Also, I'm still not sure that general anxiety thing every goes away.

**Comment by SNGmagician** [comment]
Stepping into learning something as vast as coding is a daunting task. It is frustrating, confusing, overwhelming, and even scary at times. Yet, simultaneously it is rewarding, and fulfilling. The progress you make adds together to expand your abilities. This dichotomy can be the hardest part. Luckily we wont have to go through this alone. I really am looking forward to working with all of you to expand our knowledge to create, and explore technology. Coding can be some of the most difficult languages to learn, and absolutely some of the most rewarding.

**Comment by SNGmagician** [comment]
@sweber20 
If my (admittedly limited) knowledge of coding is any indication, that preexisting knowledge will be incredibly helpful to you throughout this class. Although, I don't think the unanswered questions will ever stop. If you continue to work with code, eventually you'll be asking questions that literally no one has asked before. Through the learning process, even the youtube videos will be a great help. As those videos posed the unanswered questions, and those questions will "click" and you can explore the implications of the answer, when you find it. 
Do not discount the power of a question.

**Comment by grandfinallie** [comment]
I am very excited for this class because I have always been somewhat torn between my interests in technology and science or my love of art and creativity. This seems to be a wonderful mix of the things I am passionate about.  I am fairly new to programming, but have had a taste of what can be done with computers while working with HTML. The process can feel like an intricate puzzle that is for me to figure out. Using coding in a creative way seems like a great new challenge.

**Comment by grandfinallie** [comment]
I agree with @lindseysewell, I'm excited to see other projects! People are so different and unique, I'm sure that this class will show me some interesting perspectives different from my own.

**Comment by Audie-12** [comment]
I am afraid that I wont be able to pick up coding as fast as others and therefore I will fall behind or my projects wont turn out as well. But what I want out of the class is to be able to produce creative and complex projects and feel confident in them.

**Comment by LaneHealy** [comment]
I am eager to learn coding but am afraid that I won't be able to learn as quick as the others in the class.  On the other hand I'm very excited to be in this class with everyone!

**Comment by LaneHealy** [comment]
I'm also afraid that you are already awake and have graded these issues.  That means that I was late to turn in my first assignment and that isn't the first impression i wanted to give off.  Good morning!

**Comment by Audie-12** [comment]
Also I wasn't sure if it was like two different things in one post or you had to make two posts, so another thing is that I feel like this will be a fun and easy going class that will also help to push us as students but not to the breaking point.

**Comment by michaelmusick** [comment]
Nice job everyone! Many of you mention almost equal parts excitement and anxiety. Both of these are great to have. This is a challenging course, no point in lying about that. But it will make you better Media Artists, regardless of the paths you take. That much I promise. 

A lot of you also brought up the important connection that this is, essentially, the same as learning a language. It is, 100%! Just like learning a new language, what will make this course easier is regular practice. I would encourage you to get into the habit of spending some amount of time almost every day (6 out of 7) on coding. Just do a little bit everyday and it will get easier and better. Just like learning a language. Just like learning to ride a bike. Or just like learning a musical instrument. 

@Marahute you are wise! Yes, they are different. 

@Audie-12, Thank you for the honesty about your fears. I understand completely that worry. All you can do though is move at YOUR pace. This is not a competition, I will push you, but it will be as your are ready. 

@LaneHealy, nope, you got it in before I visited the page. Its a dangerous game, but you won this one.

**Comment by courtneythomas50** [comment]
I have always been very interested in coding and how it works. I am expecting this class to be challenging but in a good way. I am hoping to get a better understanding of what coding is and what you can create using coding. I am somewhat experienced with basic coding, but am hoping to learn so much more! I am hoping to be able to help others and receive the help and knowledge of others when needed.
I am very excited and have high hopes for this class!

**Comment by SJoCurtis** [comment]
I feel as though this class is going to be pretty good, after the two classes I have attended. Though I have minimal coding expirence, and the coding expirence I do have is with an entirely different program than the ones we will use in this class, I am eager. I am eager to learn, and become a better coder. This class does excite me as an artist and a technology/science enthusiast because of the intersection. I am always working my personal hardest to combine the two feilds in my own work. I do a lot of politcal and artisistic maps using the computer program ArcGIS and ESRIstorymaps. Im hoping a more comprehensive understanding of creative coding will allow me to begin working on projects that can combine all my passions and mediums. I know at the end of this class my coding abilities will still have a long way to grow but we all must start somewhere.

**Comment by irzahn** [comment]
<p>
I won't lie, this class seams overwhelming, but it is just the first week and I'm sure I will adapt and overcome like everything else. This class will help me by giving me the tools necessary to take on Mart-440 and will further me in the direction of becoming an artist. I will no doubtably continuously be a pain in the ass to everyone around me because I get confused easily and will probably ask the same questions more than once. Don't hold it against me please, I realize I may look like an ass hole and usually I am one but I promise to be on my best behavior and look forward to learning with you all.  <p/>
Response to Marahute, I agree with your statement about building a solid foundation,  hope I can stay grounded and focused enough to accomplish that as well.

**Comment by JConnor225** [comment]
I'm excited for this course. I dont really have any experience with coding so I'm hoping this class will give me a solid foundation and some basic experience in the field. I find it really fascinating the emergence of art through previously utilitarian mediums like code. For example, I remember Google programmed an HTML5 "music video" for The Arcade Fire that was featured on their homepage. I also really like the utilitarian use of code for building websites and applications and hope to learn some of that as well.

**Comment by JConnor225** [comment]
@SJoCurtis That's awesome that you use those programs already and create things with them, those aren't programs I was familiar with but they look really cool. That intersection between art and science you mentioned is what really interests me about this class too.

**Comment by Bswenumson** [comment]
I, like many of my classmates, am new to coding.  Reading through the syllabus I can see that there is a significant amount to learn.  The work sounds challenging, but rewarding.  I look forward to unlocking this new skill set and gaining a more comprehensive idea on the various ways to use it.

**Comment by Bswenumson** [comment]
@irzahn Don't worry about being an ass, I'm sure your questions and the repetition will only help those of us newbs around you learn faster.  I know it is going to take me a lot of study to become familiar with coding.

**Comment by kaylobro** [comment]
I'm excited to learn how to program because in the pass I never had the chance to program because I was working on building the robot then programming it but hopefully I can keep up and learn how to do it.

**Comment by ghost** [comment]
Sorry I'm late! 
I'm pretty excited for this class. The first couple have been slow but that's totally understandable because it's a technology based class. Once we all get a grip on these programs I'm sure it will speed up. I, too, have minimal coding experience and look forward to getting better at it. Here's to a good semester!

**Comment by danandcook** [comment]
I was having a lot of trouble getting my Command Prompt to work. Every command I entered responded with "this is not recognized as an internal or external command". I couldn't figure out what I was doing wrong, until I got it through my thick skull that Mac OS and Win OS do not the same commands.
I guess the lesson here is to pay attention to compatibility issues.

**Comment by michaelmusick** [comment]
Ha, Dan, that is a tough one sometime. But a good lesson, I will work to make sure I am clear about differences in macOS vs Windows 10 whenever possible. Thank you for bringing that up.

**Comment by lawlessofcourse** [comment]
This first class of Creative Coding has me excited for the new things I will learn. This semester I expect to be a lot of dedication and work. I took Intro to C++ and that took up most of my time. I'm a little nervous about learning coding again but, this class seems a lot more fun since we can be more creative with our assignments. I'm excited to see what this class will bring me.

**Comment by lawlessofcourse** [comment]
@DylanBelcourt I am also super excited to learn to code in creative ways. I took C++ when I was in Engineering it was more difficult than learning a language. But now that I can do it in a more creative and free manner I think it be a lot more fun to learn, as a result making it easier to learn.
